docs/OMS-shell-bundle.md

```diff
@@ -0,0 +1,47 @@
+### What is the OMS shell bundle?
+The OMS agent for Linux is provided in a shell script bundle which is essentially a shell script to which a binary payload is appended. This payload is a tar file that is created by copying the appropriate .deb and .rpm files collected from the specified version of omi, dsc and scx, as well as the oss-kits.
+
+### How is it built? 
+[create_bundle.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/bundle/create_bundle.sh) is the script that will create the shell bundle given an existing tar file payload. This script takes in four parameters:
+  1. Target directory - the dir path where the bundles are created
+  2. Intermediate directory - the dir path to the intermediate directory where all the intermediate targets are saved, separate from the final targets
+  3. tar file - the tar file that contains the .deb/.rpm files
+  4. Install type - the value is "DPKG" for a .deb bundle, "RPM" for an .rpm bundle, blank for an all-inclusive bundle
+  
+This script edits the bundle file i.e. [bundle_skel.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/bundle/bundle_skel.sh) and enters hard-coded values for the variables: TAR_FILE, &nbsp; OMI_PKG, &nbsp; OMS_PKG,  &nbsp;DSC_PKG,  &nbsp;SCX_PKG,  &nbsp;SCRIPT_LEN,  &nbsp;SCRIPT_LEN_PLUS_ONE  &nbsp;and &nbsp; INSTALL_TYPE. After these values are in place, the script creates the bundle in the target directory.
+
+The [Makefile](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/build/Makefile) is where the bundle creation is executed. Here, the .rpm and .deb packages are built using the installbuilder. The files from these packages along with the oss-kits are copied over to create the tar files.The create_bundle.sh script is then run to create the omsagent shell bundle in the target directory using the tar file. All bundles are put in the target directory once the build is complete.
+
+### Installing the bundle:
```

Comment:
Nicely done, just what I wanted!

I wonder if `Installing the bundle` should be documented in a more generic place (like the README file for OMS-Agent-for-Linux or Build-OMS-Agent-for-Linux), and then have this have a link to that?

Also, it's easy to change the shell bundle to include something new but forget to update this. So say something like:

`Bundle operations, at the time this document was written, is as follows. Running the bundle with the --help qualifier will show the latest set of options for shell bundles.`

Something like that, anyway.